src/data.js

```diff
@@ -31,6 +31,12 @@ genotet.data.serverURL;
 genotet.data.uploadURL;
 
 /**
+ * User queries for user info and log.
+ * @type {string}
+ */
+genotet.data.userURL;
```

Comment:
I don't think it is a good idea to combine logging and authentication as they are two separate things. We can have one URL for authentication and another for logging.